btree/btree.c

```diff
@@ -3080,18 +3146,13 @@ static struct m0_sm_state_descr btree_states[P_NR] = {
 	[P_CHECK] = {
 		.sd_flags   = 0,
 		.sd_name    = "P_CHECK",
-		.sd_allowed = M0_BITS(P_MAKESPACE, P_CLEANUP, P_DOWN, P_ACT,
-				      P_INIT, P_NEXTUP),
+		.sd_allowed = M0_BITS(P_MAKESPACE, P_CLEANUP, P_DONE, P_DOWN,
+				      P_ACT, P_INIT),
 	},
 	[P_MAKESPACE] = {
 		.sd_flags   = 0,
 		.sd_name    = "P_MAKESPACE",
-		.sd_allowed = M0_BITS(P_ACT, P_NEXTUP),
-	},
-	[P_NEXTUP] = {
-		.sd_flags   = 0,
-		.sd_name    = "P_NEXTUP",
-		.sd_allowed = M0_BITS(P_MAKESPACE, P_CLEANUP),
+		.sd_allowed = M0_BITS(P_CLEANUP, P_DONE, P_ACT),
```

Comment:
Not jumping to P_DONE from P_MAKESPACE, this is not allowed state.

Reply:
I added it because if we use return m0_sm_op_sub(&bop->bo_op, P_CLEANUP, P_DONE); instead of P_CLEANUP, so I thought transition P_MAKESPACE to P_DONE will be needed.